src/agents/customer_group.jl

```diff
@@ -341,51 +358,77 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         ),
     )
 
-    green_sub_model = GreenSubModel(
-        ParamArray(
-            "Constant",
-            (model_data.index_h,),
-            [0.0, 0.0, 0.0]; 
-            description = "Constant in green power uptake function (regression parameter)",
+    function generate_coefficients(index_h, h_to_sector, coefficients)
+        return [
+            get(coefficients, h_to_sector[index], 0.0) for index in index_h
```

Comment:
nit: Elsewhere in the code we would write `for h in index_h` instead of `for index in index_h`.